src/elements/Button/ButtonGroup.tsx

```diff
@@ -0,0 +1,38 @@
+import React, { FC, useMemo } from 'react';
+import {
+  ButtonGroupContext,
+  ButtonGroupContextProps
+} from './ButtonGroupContext';
+
+export interface ButtonGroupProps extends ButtonGroupContextProps {
```

Comment:
I would add the `onChange` and the `selected` value probably. How does MUI / others do this?

Reply:
[MUI](https://mui.com/material-ui/api/button-group/#props) and others don't have these props and I think it makes sense too as unlike a `RadioGroup`, a `ButtonGroup` just groups the buttons into a unified display so it does not need to have a `selected` value or a `onChange` prop.